test/e2e/specs/wp-calypso-gutenberg-editor-tracking-spec.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import assert from 'assert';
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import LoginFlow from '../lib/flows/login-flow.js';
+
+import GutenbergEditorComponent from '../lib/gutenberg/gutenberg-editor-component';
+import WPAdminSidebar from '../lib/pages/wp-admin/wp-admin-sidebar';
+
+import * as driverManager from '../lib/driver-manager.js';
+import * as dataHelper from '../lib/data-helper.js';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+const screenSize = driverManager.currentScreenSize();
+const host = dataHelper.getJetpackHost();
+
+// We need to trigger test runs against Gutenberg Edge (the "next" version of Gutenberg that
+// will be deployed to Dotcom) as well as the current version of Gutenberg.
+const gutenbergUser =
+	process.env.GUTENBERG_EDGE === 'true' ? 'gutenbergSimpleSiteEdgeUser' : 'gutenbergSimpleSiteUser';
+
+let driver;
+
+before( async function () {
+	this.timeout( startBrowserTimeoutMS );
+	driver = await driverManager.startBrowser();
+} );
+
+function getEventsFiredForBlock( eventsStack, event, block ) {
+	if ( ! eventsStack || ! event || ! block ) {
+		return false;
+	}
+
+	return eventsStack.filter(
+		( [ eventName, eventData ] ) => event === eventName && eventData.block_name === block
+	);
+}
+
+function getTotalEventsFiredForBlock( eventsStack, event, block ) {
+	return getEventsFiredForBlock( eventsStack, event, block ).length;
+}
+
+describe( `[${ host }] Calypso Gutenberg Tracking: (${ screenSize })`, function () {
+	this.timeout( mochaTimeOut );
+
+	describe( 'Tracking: @parallel', function () {
+		// Create image file for upload
+		before( async function () {} );
+
+		step( 'Can log in to WPAdmin and create new Post', async function () {
+			this.loginFlow = new LoginFlow( driver, gutenbergUser );
+			if ( host !== 'WPCOM' ) {
+				this.loginFlow = new LoginFlow( driver );
+			}
+			// await this.loginFlow.loginAndStartNewPage();
```

Comment:
I'm guessing this and [`loginAndStartNewPost` function](https://github.com/Automattic/wp-calypso/blob/eed5c383d86a89c04b96300d5de856457fb03d96/test/e2e/lib/flows/login-flow.js#L103) don't work since you have to do the stuff from `wp-admin`? 

If that's all the case, maybe remove this commented out line?